charts/illumidesk/templates/grader_setup_service.yaml

```diff
@@ -113,8 +113,13 @@ spec:
             - containerPort: 8000
           env:
             - name: GRADER_IMAGE_NAME
-              value: {{ .Values.graderSetupService.graderImage }}
-              
+              value: {{ .Values.graderSetupService.graderSpawnerImage | default "illumidesk/grader-notebook:latest" }}
+            - name: GRADER_CPU
```

Comment:
Based on the comments above, update these to:

- GRADER_REQUEST_CPU
- GRADER_REQUEST_MEM
- GRADER_LIMIT_CPU
- GRADER_LIMIT_MEM